libpod/info_test.go

```diff
@@ -0,0 +1,59 @@
+package libpod
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/containers/podman/v4/libpod/define"
+	"github.com/stretchr/testify/assert"
+)
+
```

Comment:
Nit: We could also test `getCPUUtilization()` to make sure it doesn't error out and that it's returning anything between [0,100]